wkz/watchdogs.py

```diff
@@ -31,7 +31,10 @@ def trigger_device_watchdog():
 def _watch_for_device(path_to_garmin_device: str, path_to_trace_dir: str, delete_files_after_import: bool):
     device_mounted = False
```

Comment:
`device_mounted` seems pretty useless in this function. Probably forgot to clean this up after some refactoring. If you want you could remove the usage of this variable. But no need to do it within this PR - I can take care of it, too.

Reply:
I guess that would simply be removed all mentions of `device_mounted` in this function right?

Reply:
exactly